docs/Reference/Request.md

```diff
@@ -42,6 +42,7 @@ Request is a core Fastify object containing the following fields:
   handling the request
 - `routeConfig` - The route [`config`](./Routes.md#routes-config) 
   object.
+- `routeBodyLimit` - either server limit or route limit.
```

Comment:
as described in the issue, here we should add a:

```
request.routeOptions.bodyLimit
```

Reply:
Thanks for Review @Eomm 

Should I include `routeBodyLimit` as well as `routeOptions` property or just replace `routeBodyLimit` with `routeOptions`